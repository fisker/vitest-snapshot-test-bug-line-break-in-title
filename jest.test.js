test('CR\r', () => {
  expect(1).toMatchSnapshot()
})

test('CRLF\r\n', () => {
  expect(2).toMatchSnapshot()
})