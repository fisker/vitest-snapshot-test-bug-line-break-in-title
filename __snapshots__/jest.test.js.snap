// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`CR 1`] = `1`;

exports[`CRLF
 1`] = `2`;
